File: Functional Programming/Tugas_praktikum/filter.dart
// bilangan kelipatan 2

void main(){
  List<int> listAngka = [1,2,3,4,5,6,7,8,9,10];
  final kelipatanDua = listAngka.where((listAngka) => listAngka % 2 == 0);
  print(kelipatanDua);
}

File: Functional Programming/Tugas_praktikum/filter.py
# bilangan kelipatan 2

def kelipatandua(a):
  if a % 2 == 0 :
    return a
  
list_angka = [1,2,3,4,5,6,7,8,9,10]

filter_list = list(filter(kelipatandua, list_angka))
print(filter_list)
